app/views/docs/functions.phtml

```diff
@@ -384,6 +384,35 @@ func main(req: RequestValue, res: RequestResponse) throws -> RequestResponse {
     <p>If you're running Appwrite locally and trying to connect to your local Appwrite instance from a cloud function, you'll have to use your local IP address in place of localhost. This is because, within the container localhost does not refer to the host machine rather the container itself. </p>
 </div>
 
+<h2><a href="/docs/functions#ignoreFiles" id="ignoreFiles">Ignore Files</a></h2>
```

Comment:
Let's move this section to the end of this page.

Right now, it disturbs the flow of the page which goes from 
1. Quick start
2. Writing your own function
3. Creating your function and so on,...

Ignoring files can be further down in the docs

Reply:
![Screenshot 2022-05-08 at 7 20 00 PM](https://user-images.githubusercontent.com/20852629/167303063-c9b68c27-715c-4232-9714-d410828f18ce.png)

Before Supported Runtimes can be a good spot for this.

Reply:
Agree. Moved above supported runtimes.